core/columnar/query_component.cxx

```diff
@@ -59,6 +60,7 @@ class pending_query_operation
                           std::chrono::milliseconds default_timeout)
     : client_context_id_{ uuid::to_string(uuid::random()) }
     , timeout_{ options.timeout.value_or(default_timeout) }
+    , payload_(build_query_payload(options))
```

Comment:
Using the initializer list syntax for `tao::json::value` resulted in the `error: no class template named 'default_key'` error on Linux that we've seen before - unsure why